plone/app/contenttypes/migration/field_migrators.py

```diff
@@ -17,17 +17,24 @@ def migrate_simplefield(src_obj, dst_obj, src_fieldname, dst_fieldname):
     to the target-object. The only transform is a safe_unicode of the value.
     """
     field = src_obj.getField(src_fieldname)
-    if field:
-        at_value = field.get(src_obj)
+    try:
+        if field:
```

Comment:
Can we decrease the amount of code inside the try/except?

Reply:
Yeah, I thought about that too.  It's a trade off, more code inside the try/except or *2* try/excepts, some repetition of code, and many more lines of code.  I thought the risk of something unintended in the try/except raising `AttributeError` wasn't worth the sacrifice in readability.

Reply:
Ok, I see your point :)